Dockerfile.ubi

```diff
@@ -202,7 +202,7 @@ FROM vllm-openai as vllm-grpc-adapter
 USER root
 
 RUN --mount=type=cache,target=/root/.cache/pip \
-    pip install vllm-tgis-adapter==0.5.1
+    pip install vllm-tgis-adapter==0.5.2
```

Comment:
Could you also update `Dockerfile.rocm.ubi`? thanks!

Reply:
Done